### ISSUE TITLE
Pyvista issue with volume data representation

### ISSUE THREAD

**Issue by xhlulu** [plotly/dash-vtk#55]
See readme https://github.com/lizzjs/dash-vtk

@jourdain this seems out of my domain, do you think this is an easy fix or something that can be added to docs?

**Comment by jourdain** [comment]
I'm wondering if it is related to numpy array. I'm currently off but I would try to add a `tolist()` or something along that to work around the current issue on [that line](https://github.com/lizzjs/dash-vtk/blob/main/test_pv_dashvtk.py#L49) assuming the issue is related to that part.

**Comment by lizzjs** [comment]
> I'm wondering if it is related to numpy array. I'm currently off but I would try to add a `tolist()` or something along that to work around the current issue on [that line](https://github.com/lizzjs/dash-vtk/blob/main/test_pv_dashvtk.py#L49) assuming the issue is related to that part.

I tried what you suggested and it still gives the same error.

**Comment by jourdain** [comment]
Maybe I should have asked what is the error?

**Comment by jourdain** [comment]
But my guess is that you want geometry rendering with glyphing or something along those lines. But maybe I'm wrong and I have no idea on what you are trying to see.

**Comment by jourdain** [comment]
Maybe some stream tracer like with the bike demo?

![test](https://user-images.githubusercontent.com/294518/125113673-3008ad00-e0a6-11eb-8573-b91cb51a228b.jpg)

**Comment by lizzjs** [comment]
I believe this is exactly what we want, to achieve this you used geometry rendering?

**Comment by jourdain** [comment]
Yes, geometry rendering with some filtering to generate the stream lines. (sorry for the late reply, just got back from vacation)

**Comment by lizzjs** [comment]
I think I am a bit confused on how to render it using dash-vtk, are you using PyVista or VTK to load the data points? (Also, it's okay, thank you for your reply)

**Comment by jourdain** [comment]
To me VTK or PyVista is the same thing. That only affect the syntax but under the hood it is VTK.
From your grid you need to do a stream tracer or something similar (look at the motorcycle example) and then with dash-vtk you just need to use a geometry representation with a mesh.

HTH
